website/pages/blog/auditing-service-account-keys-with-cloudquery.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Auditing Service Account Keys with CloudQuery
+tag: security
+date: 2022/11/04
+description: >-
+  A case against Service Account Keys.
+author: SCKelemen, Alex
+---
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+
+# Intro
+There are several common methods of accessing cloud 
+environments, each with their own risks and uses. Service 
```

Comment:
Curious - do Service Account Keys refer to GCP Service Account Keys or Keys in general across Cloud Providers?